external/llvm-project/mlir/lib/Dialect/Async/IR/Async.cpp

```diff
@@ -53,6 +54,174 @@ YieldOp::getMutableSuccessorOperands(Optional<unsigned> index) {
   return operandsMutable();
 }
 
+//===----------------------------------------------------------------------===//
+/// LaunchOp
+//===----------------------------------------------------------------------===//
+
+void LaunchOp::build(OpBuilder &builder, OperationState &result, func::FuncOp func,
```

Comment:
Can this more generally be "any callable"/"a symbol", or does it need to be the func opp itself?

Reply:
for now a func op. (no indirection)